### DISCUSSION TITLE
mcu board

### DISCUSSION THREAD

**Discussion by Podster2020** [3DCoded/3MS#13] [Configuration Help]
i have a octopus v1.1, geeetech a10 gt2560 v4, btt mmu, can i use any of these for setting up 3MS?

can i just change the pin outs to suit

**Comment by 3DCoded** [comment]
Currently support is implemented for the Geetech A30T. I can work on support for the Octopus v1.1 or the BTT MMU. Do you prefer either one? It looks like the BTT MMU board would support four filaments, and the BTT Octopus board would support eight filaments.

**Reply by Podster2020** [reply]
Can you do the btt mmu if you can, as it's a more compact board and 4 will be enough the board has loads of sensors

**Reply by 3DCoded** [reply]
I've added support and documentation for the BTT MMB.
